.github/workflows/show-map-diffs.yml

```diff
@@ -0,0 +1,34 @@
+name: Map diffs
+
+on: [pull_request]
```

Comment:
If you use `pull_request_target` we can use secrets for PRs from forks

Reply:
```suggestion
on: [pull_request_target]
```